docs/docs/gatsby-on-windows.md

```diff
@@ -16,3 +16,7 @@ powershell console. On installing this package, it downloads and installs the
 Visual C++ Build Tools 2015, provided free of charge by Microsoft. These tools
 are required to compile popular native modules. It will also install Python
 2.7, configuring your machine and npm appropriately.
+
+## Installing dependencies without troubles.
```

Comment:
Looks like this is what you need to do to get Hyper setup https://github.com/zeit/hyper/pull/946#issuecomment-257201607

Perhaps just add this to the instructions instead of this section?